### DISCUSSION TITLE
Upgrade to Monterey Recommendation

### DISCUSSION THREAD

**Discussion by varunpan** [tylernguyen/x1c6-hackintosh#155] [Q&A] [answered]
Do I need to create a USB disk like I did for the fresh install for Big Sur or can I upgrade using the System preferences and then just update the EFI? 

What is the recommended process? Will I lose all my data if I need to upgrade using USB?

**Comment by tylernguyen** [comment, marked as answer]
@varunpan 

Regular upgrade via System Preferences should work.

However, I recommend creating a installer USB and doing a fresh install of macOS Monterey. You will lose all data with this method but the experience should be more stable.

**Reply by varunpan** [reply]
Thank you for your response. Should I update the EFI before the upgrade from system preferences? (I really don't want to lose all my data). Also, should I use the same CPUfriend.kext, and intel bluetooth/wifi kext as well, or do I need to update that after the upgrade?
